kai/reactive_codeplanner/main.py

```diff
@@ -16,17 +19,26 @@
 from kai.reactive_codeplanner.task_runner.compiler.maven_validator import (
     MavenCompileStep,
 )
+from kai.reactive_codeplanner.task_runner.dependency.task_runner import (
+    DependencyTaskRunner,
+)
 from kai.reactive_codeplanner.vfs.git_vfs import RepoContextManager
 
 # Configure logging
-logging.basicConfig(level=logging.INFO)
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Why this change?

Reply:
Cruft from before I rebased on the logging PR

Reply:
I think we may just want to remove this line then?